### ISSUE TITLE
What exact time is morning?

### ISSUE THREAD

**Issue by sleeperzjf** [chubin/wttr.in#462]
`morning noon evening night`

**Comment by chubin** [comment]
Mapping used in the v1 view:

```
Morning:     9:00
Noon:       12:00
Evening:    18:00
Night:      22:00
```

**Comment by chubin** [comment]
We already had issues with such questions before, so I think this info should be added to the help

**Comment by sleeperzjf** [comment]
I think it will be easier for users just to add the time range below `morning` or other terms.

**Comment by iago-lito** [comment]
I'm surprised, as I thought these were *periods* rather than *instants*. In french, they show up as

![periods](https://user-images.githubusercontent.com/10091387/83950427-0f2eb200-a82b-11ea-8aed-f9bead4d915b.png)

which rather translates as `morning | afternoon | evening | night`. I found it was clear, but now that I've read this v1 mapping, I don't understand anymore X)

**Comment by sleeperzjf** [comment]
I'm a little confused too, 'cause I'm writing a script that automatically gets `highest temperature`, `lowest temperature`, and `chance of rain` from the `v1 view` and shows them in my status bar.
But if `morning noon evening night` just means ***instants***, then it won't work that way.

**Comment by chubin** [comment]
Yes, you are right. This output is a little bit confusing indeed, and probably
we should change it to reflect time ranges instead of time instants.
Something like:

```
Morning:     8:00-12:00
Afternoon:       12:00-18:00
Evening:    18:00-22:00
Night:      22:00-8:00
```

temperature range (`MIN..MAX`) will then show the max and the min temperatures
for the interval, and the rest — average values.

Would it be better?

**Comment by sleeperzjf** [comment]
Well, I think it will be better, but it's just me, I don't know about other people.

**Comment by bend-n** [comment]
what about just using the actual times, 9 12 18 22 (or 6 10)?
